### PR TITLE
ulong typedef has been removed in PHP 7.4

### PR DESCRIPTION
**Pull request by ttraeger**
see https://raw.githubusercontent.com/php/php-src/PHP-7.4/UPGRADING.INTERNALS

### CODE REVIEW

**Comment by stefnats** [comment]
I stumble upon this bug as well when manually compiling (which is necessary for 7.3+).
However, this PR would only fix this for PHP `^7.4` and not 8.0 or above.
Why not change line 120 to

```
#if ((PHP_MAJOR_VERSION == 7 && PHP_MINOR_VERSION >= 4) || PHP_MAJOR_VERSION > 7)
```

File: ext/php_driver.h [hunk 1]
```diff
@@ -117,6 +117,9 @@ typedef zend_object php5to7_zend_object_free;
 typedef zval **php5to7_zval_gc;
 typedef zval *php5to7_dtor;
 typedef size_t php5to7_size;
+#if PHP_MAJOR_VERSION >= 7 && PHP_MINOR_VERSION >= 4
```
Comment (stefnats) [inline ext/php_driver.h]: Thanks for the fix! I stumble upon this bug as well when manually compiling (which is necessary for 7.3+).
However, this PR would only fix this for PHP ^7.4 and not 8.0 or above.
Why not change line 120 to
```
#if ((PHP_MAJOR_VERSION == 7 && PHP_MINOR_VERSION >= 4) || PHP_MAJOR_VERSION > 7)
```

Comment (ttraeger) [reply ext/php_driver.h]: Thanks for the hint, I changed the PR as you suggested.

**Comment by mpenick** [comment]
Thanks for the PR!

**Comment by stefnats** [comment]
@mpenick currently installing the php cassandra extension is a real pain. When do you think will you tag a new release? Thanks in advance

**Comment by adutra** [comment]
@stefnats We are preparing a new release, should take us a few weeks. Stay tuned!

**Comment by eliasmajic** [comment]
@adutra curious if there is an update

**Comment by adutra** [comment]
@eliasmajic unfortunately, as I said earlier today on the mailing list, we are not anymore in capacity of resuming the development on the C++ and PHP drivers.

I know how this must be disappointing for you all, but that's the way it is right now. If you can build the driver on your own, that would be your best option currently.

**Pull request merged**